tools/acrntrace/README.rst

```diff
@@ -33,6 +98,14 @@ Usage
 
       q <enter>
 
+#. Convert trace data to human-readable format, with:
+   .. code-block:: none
```

Comment:
add a blank line before this code-block directive

Reply:
Will do. Thanks, David